src/controllers/trip.controller.js

```diff
@@ -0,0 +1,35 @@
+import TripService from '../services/trip.service';
+import AuthenticationService from '../services/authentication.service';
+import responseHandlers from '../utils/responseHandlers';
+import statusCodes from '../utils/statusCodes';
+import customMessages from '../utils/customMessages';
+import utils from '../utils/authentication.utils';
+
+const { updatedResponse, errorResponse } = responseHandlers;
+const { decodeToken } = utils;
+const { createReturnTrip } = TripService;
+const { findUserEmailIfExist } = AuthenticationService;
```

Comment:
Same thing here too `findUserEmailIfExist` is not being used in this file.

Reply:
I refactored and they are no longer there
because we will be using same controller,route and also service